### ISSUE TITLE
Hapus Modul Covid19

### ISSUE THREAD

**Issue by andifahruddinakas** [OpenSID/OpenSID#8633]
Hapus Modul Covid19
![image](https://github.com/user-attachments/assets/dd43ebea-c8be-4389-8bbf-5cef1e14faf4)

Modul yang dihapus sbb:
- [ ] Pendataan
- [ ] Vaksin
- [ ] Pemantauan

**Comment by eddieridwan** [comment]
@andifahruddinakas penerapan penghapusan modul Covid19 harus menunggu pembahasan dan pemberitahuan pada komunitas dulu. Lakukan analisisnya dulu, memastikan sudah siap melakukan kalau lampu hijau telah diberikan.

**Comment by andifahruddinakas** [comment]
> @andifahruddinakas penerapan penghapusan modul Covid19 harus menunggu pembahasan dan pemberitahuan pada komunitas dulu. Lakukan analisisnya dulu, memastikan sudah siap melakukan kalau lampu hijau telah diberikan.

Iya, memang belum dikerjakan dan tidak masuk target. Sudah dilabeli **analisis** yang memerlukan pengecekan terlebih dahulu.

**Comment by husnulseptia2209** [comment]
apakah ini mau dilanjutkan pak @eddieridwan , @andifahruddinakas ?

**Comment by eddieridwan** [comment]
Ya, nanti perlu dihapus, karena sudah tidak digunakan lagi. Tapi tunggu dulu, karena perlu disosialisasikan dulu. Dan perlu ada solusi bagi desa yg ingin mengarsipkan datanya.

**Comment by andifahruddinakas** [comment]
Cek pembahasan https://github.com/OpenSID/OpenSID/issues/10196#issuecomment-3354529501
